js/script.js

```diff
@@ -1,10 +1,12 @@
-function startTime() {
-  var today = new Date();
-  var h = today.getHours().toString().padStart(2, '0');
-  var m = today.getMinutes().toString().padStart(2, '0');
-  var s = today.getSeconds().toString().padStart(2, '0');
-  document.getElementById('time').innerHTML = `${h}:${m}:${s}`;
-  window.setTimeout(() => startTime(), 500);
-}
-
-startTime();
\ No newline at end of file
+(function() {
+  var time = document.getElementById('time');
```

Comment:
As @lempiy noted, think about caching instead of hitting DOM every half of second.